docs/mutation-void.md

```diff
@@ -0,0 +1,25 @@
+Returning Void Type in Mutations with SmallRye GraphQL
+=======
+While by default GraphQL specification doesn't provide direct support for returning `Void` type from mutation operations, SmallRye GraphQL has implemented this feature based on user feedback.
+
+To implement a mutation operation that returns `Void` type, you can use the following pattern in your annotated `@GraphQLApi` class:
+```java
+    @Mutation
+    public Void createHero(SuperHero hero) {
+        superHeroService.create(hero);
+        return null;
+    }
+```
+In this example, createHero() receives a SuperHero object as input,
+performs some operation with it (in this case, creating it using a
+superHeroService), and then returns null.
+
+> [NOTE]
```

Comment:
I understand that it would be not easy to do, but returning the primitive `void` would be much more Java-like... i.e. so much cooler! Maybe you can take a deeper look again?

Reply:
> I understand that it would be not easy to do, but returning the primitive `void` would be much more Java-like... i.e. so much cooler! Maybe you can take a deeper look again?

I went by the Phillip's comment: https://github.com/quarkusio/quarkus/discussions/25381#discussioncomment-2690523 But I agree with you. Will have a look at it.